src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Created By: Chris Heathe
+ * Created On:  October28, 2017
+ * Description: A node that will theoretically interpret lidar data,
+ *              and help Olaf navigate through a hole in a row of cones
+ *              Stress on *theoretically*
+ */
+
+#ifndef obstacle_avoidance_MYNODE_H
+#define obstacle_avoidance_MYNODE_H
```

Comment:
We should rename these header guards to something like `OBSTACLE_AVOIDANCE_H`. If you're not familiar with header guards, this stack overflow question has some great explanations: https://stackoverflow.com/questions/4767068/header-guards-in-c